libraries/AP_Scripting/docs/docs.lua

```diff
@@ -3906,6 +3906,14 @@ function fence:get_breach_time() end
 ---| 8 # Minimum altitude
 function fence:get_breaches() end
 
+-- Returns the type bitmask of any fence whose margins have been crossed
+---@return integer fence_type bitmask
+---| 1 # Maximim altitude
+---| 2 # Circle
+---| 4 # Polygon
```

Comment:
It might be helpful to know it it was an include or exclude fence.

Reply:
Not easy to do and we don't currently do that for breaches